R/writeTimeseries.R

```diff
@@ -69,32 +66,31 @@ writeTimeSeries <- function(tl,
       if(!wide) {
         # convert the list into a pretty, long data.frame
         if(data.table_available){
-          tl_lengths <- sapply(tl, length)
+          tl_lengths <- data.table(length = sapply(tl, length))
```

Comment:
You need to call **data.table::data.table** because we don't import data.table (if we did we had a dependency. Checking that data.table is available (is installed) doesn't mean it's important.